src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -39,6 +39,8 @@ FixedwingPositionControl *l1_control::g_control;
 static int _control_task = -1;			///< task handle for sensor task */
 
 FixedwingPositionControl::FixedwingPositionControl() :
+	_sub_airspeed(ORB_ID(airspeed), 0, 0, nullptr),
```

Comment:
If airspeed is set to disabled it should not be used. the sensor might be bad or simply mounted for reference data. the check to apply should be done on the param, not on the valid state